lms/djangoapps/courseware/module_render.py

```diff
@@ -642,6 +643,8 @@ def rebind_noauth_module_to_user(module, real_user):
 
     system.set(u'user_is_staff', user_is_staff)
     system.set(u'user_is_admin', has_access(user, u'staff', 'global'))
+    system.set(u'user_is_beta', CourseBetaTesterRole(course_id).has_user(user))
+    system.set(u'days_early_for_beta', getattr(descriptor, 'days_early_for_beta', 0))
```

Comment:
I am thinking why don't we pass the number of seconds here instead of the number of days? So if in the future we make this setting more granular than a day, xblocks will not need to be updated? 

@adampalay do you have any thoughts?